src/CMakeLists.txt

```diff
@@ -549,7 +552,10 @@ if(NOT CEPH_BUILD_VIRTUALENV)
   set(CEPH_BUILD_VIRTUALENV /tmp)
 endif()
 
+if(WITH_PYTHON)
 add_subdirectory(pybind)
+endif()
+
```

Comment:
I was hoping not chase down every use of python in the tree, but instead just assume that the user can call a specific target when invoking make that does not require python, for example

```
mkdir build && cd build
cmake .. -DWITH_PYTHON=OFF
make ceph-mon ceph-osd
```

As you point out python is assumed throughout the code base and it would be difficult to chase it all down.